Additional Features > Reasoning (#261)
agno-agi/docs: concepts/reasoning/reasoning-agents.mdx

---
title: Reasoning Agents
sidebarTitle: Overview

> description: Reasoning Agents are a new type of multi-agent system developed by Agno that combines chain of thought reasoning with tool use.

Reviewer: ```suggestion
description: Reasoning Agents enables multi-agent systems that combine chain of thought reasoning with tool use.
```
